tiup/tiup-cluster-no-sudo-mode.md

```diff
@@ -0,0 +1,204 @@
+
+## Prepare the user and configure the SSH mutual trust
+
+1. Log in to all deployment target machines in sequence, and use the `root` user to create a user named `tidb` with the following command. In no-sudo mode, there is no need to configure password-free sudo for the `tidb` user, that is, you do not need to add the `tidb` user to sudoers.
+   
+    ```shell
+    adduser tidb
+    ```
+   
+2. Start the `systemd` user mode for the `tidb` user on every deployment target machine. This step is required and do not skip it.
+
+    1. Use the `tidb` user to set the `XDG_RUNTIME_DIR` environment variable.
+   
+       ```shell
+       mkdir -p ~/.bashrc.d
+       echo "export XDG_RUNTIME_DIR=/run/user/$(id -u)" > ~/.bashrc.d/systemd
+       source ~/.bashrc.d/systemd
+       ```
+   
+    2. Use `root` user to start user service.
+   
+        ```shell
+        $ systemctl start user@1000.service # `1000` is the ID of the tidb user. You can get the user ID by executing the `id` command.
+        $ systemctl status user@1000.service
+        user@1000.service - User Manager for UID 1000
+        Loaded: loaded (/usr/lib/systemd/system/user@.service; static; vendor preset>
+        Active: active (running) since Mon 2024-01-29 03:30:51 EST; 1min 7s ago
+        Main PID: 3328 (systemd)
+        Status: "Startup finished in 420ms."
+        Tasks: 6
+        Memory: 6.1M
+        CGroup: /user.slice/user-1000.slice/user@1000.service
+                ├─dbus.service
+                │ └─3442 /usr/bin/dbus-daemon --session --address=systemd: --nofork >
+                ├─init.scope
+                │ ├─3328 /usr/lib/systemd/systemd --user
+                │ └─3335 (sd-pam)
+                └─pulseaudio.service
+                  └─3358 /usr/bin/pulseaudio --daemonize=no --log-target=journal
+        ```
+   
+    Execute `systemctl --user` in the terminal. If no errors are reported, it indicates that it has started normally.
+
+3. Use ssh-keygen in the central control computer to generate a key and copy the public key to other deployment machines.
+
+## Prepare the topology file
+
+1. Use the following command to generate topology file.
+
+    ```shell
+    tiup cluster template > topology.yaml
+    ```
+   
+2. Edit the topology file.
+
+     Compared with the previous mode, TiUP in no-sudo mode requires adding `systemd_mode: "user"` in the global block of `topology.yaml`. The `systemd_mode` parameter specifies whether to use systemd's user mode. If this parameter is not set, the default value is `system`, meaning sudo permissions are required. Additionally, in no-sudo mode, `/data` cannot be used as the `deploy_dir` or `data_di`r due to potential permission issues. You must select a path accessible to non-root users.
+   
+    The following example uses relative paths and the final paths used are `/home/tidb/data/tidb-deploy` and `/home/tidb/data/tidb-data`. The rest is the same as the old version.
```

Comment:
```suggestion
     Compared with the previous mode, TiUP in no-sudo mode requires adding `systemd_mode: "user"` in the global block of `topology.yaml`. The `systemd_mode` parameter specifies whether to use systemd's user mode. If this parameter is not set, the default value is `system`, meaning sudo permissions are required. 
     
     Additionally, in no-sudo mode, because the non-root user `tidb` has no permissions, `/data` cannot be used as the `deploy_dir` or `data_dir`. You must select a path accessible to non-root users. The following example uses relative paths and the final paths used are `/home/tidb/data/tidb-deploy` and `/home/tidb/data/tidb-data`. The rest is the same as the old version.
```